modules/rubiconBidAdapter.js

```diff
@@ -516,11 +516,11 @@ export const spec = {
     }
 
     const bidFpd = {
-      user: bidRequest.params.visitor || {},
-      context: bidRequest.params.inventory || {}
+      user: {...bidRequest.params.visitor} || {},
+      context: {...bidRequest.params.inventory} || {}
     };
 
-    if (bidRequest.params.keywords) bidFpd.context.keywords = bidRequest.params.keywords;
+    if (bidRequest.params.keywords) bidFpd.context.keywords = (utils.isArray(bidRequest.params.keywords)) ? bidRequest.params.keywords.join(',') : bidRequest.params.keywords;
```

Comment:
I thought the plan was to not support the string or non array case in bidParams.

```
if (utils.isArray(bidRequest.params.keywords)) bidFpd.context.keywords = bidRequest.params.keywords.join(',');
```

Reply:
It was but then I reviewed the existing tests and one test explicitly states that it should not error if set as a string. I dont like it but will assume there is a reason for the test that could break things in production. So for the time being I made sure this allows for both